lib/logflare_web/controllers/api/fallback_controller.ex

```diff
@@ -17,7 +17,9 @@ defmodule LogflareWeb.Api.FallbackController do
     |> halt()
   end
 
-  def call(conn, nil) do
+  def call(conn, nil), do: call(conn, {:error, :not_found})
```

Comment:
I would add task/todo comment to hunt occurrences of that and replace it with `{:error, :not_found}` tuple and remove this case.